sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLTab.scala

```diff
@@ -18,15 +18,22 @@
 package org.apache.spark.sql.execution.ui
 
 import org.apache.spark.internal.Logging
+import org.apache.spark.sql.execution.streaming.StreamQueryStore
+import org.apache.spark.sql.streaming.ui.{StreamingQueryPage, StreamingQueryStatisticsPage}
 import org.apache.spark.ui.{SparkUI, SparkUITab}
 
-class SQLTab(val sqlStore: SQLAppStatusStore, sparkUI: SparkUI)
+class SQLTab(
+    val sqlStore: SQLAppStatusStore,
+    val streamQueryStore: Option[StreamQueryStore],
```

Comment:
Do we need to use `Option` here?

Reply:
The information is only available in running app - passing None would simplify pruning the case. I've suggested to put information to SQLAppStatusStore - it may be addressed in follow-up PR and we can completely remove this.